packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -55,11 +50,33 @@ const projectButtonStyle = css`
   padding-bottom: 16px;
 `;
 
-function isProject(
-  props: ProjectMongoSelectProps | OrganizationMongoSelectProps,
-): props is ProjectMongoSelectProps {
-  return props.variant === 'project';
-}
+const nameStyle = css`
+  font-size: 14px;
+  color: ${uiColors.gray.dark3};
+`;
+
+const productStyle = css`
+  font-size: 12px;
+  color: ${uiColors.gray.dark2};
+  font-weight: bolder;
+  white-space: nowrap;
+`;
+
+const orgOptionContainerStyle = css`
+  display: flex;
+  justify-content: space-between;
+`;
+
+const formatPlanType = (planType: PlanType) => {
```

Comment:
We could simplify this by having an enum and consuming it directly:
```ts
const formattedPlanTypes: Record<PlanType, string> = {
  [PlanType.Atlas]: 'Atlas',
  [PlanType.Cloud]: 'Cloud Manager',
  [PlanType.OnPrem]: 'Ops Manager',
} as const;

const formattedAtlasType = formattedPlanTypes[PlanType.Atlas];
```

Reply:
Only benefit to function is easier behavior for default case, but yeah either works for me. The return for OnPrem looks like a bug there too.

Reply:
👍